uat/custom-components/src/main/java/com/aws/greengrass/artifacts/logGeneratorDiffName.java

```diff
@@ -0,0 +1,98 @@
+/*
```

Comment:
Why that name `LogGeneratorDiffName.java`? You already deleted the previous file

Reply:
There will be another custom component that rotate files with the same name, basically using another logger rolling policy. I am trying to make them to be 2 components instead of 1.